src/components/record-modal/record-modal.css

```diff
@@ -106,7 +106,8 @@
     text-align: center;
 }
 
-.record-button {
+/* Increase specificity to make sure this wins out over normalize.css */
+svg.record-button {
```

Comment:
This is unrelated to the feature. It was brought to my attention that the expanding circle when recording audio is being clipped to a square region. This appears to be because of some css normalization when the editor is used in `scratch-www`. This increases the specificity, overriding the rule in `scratch-www`. Sorry for sneaking in unrelated work, but it was a simple fix.

<img width="345" alt="Screenshot 2023-03-23 at 10 51 06 AM" src="https://user-images.githubusercontent.com/4893763/227241974-9dee7c5c-12aa-4ba3-bba7-6507f20d2409.png">